circuits-circom/venmo_receive.circom

```diff
@@ -7,10 +7,7 @@ include "./regexes/venmo_payer_id.circom";
 include "./regexes/venmo_timestamp.circom";
 
 template VenmoReceiveEmail(max_header_bytes, max_body_bytes, n, k, pack_size) {
-    assert(max_header_bytes % 64 == 0);
```

Comment:
Any reason to get rid of these?

Reply:
All of this has been added to the imported template now.